.travis.yml

```diff
@@ -9,11 +9,20 @@ script: tox
 matrix:
   include:
     - python: "2.7"
-      env: TOXENV=py27
+      env:
+        - TOXENV=py27
+        - secure: TnI2dSGBAF6cE2iVnE7XQC/V50pHf2nwdHV1iTtEkfFp0DNeb8xrCwyML3SSXroEEn+HgVs6Jg+Xg9G5axo4A8EI3GE+COuP1E2Y10I7lutjf9xPep1VmFS13+xGBg9zX5xy7cSG33RgIaxhk6CLnBo5b8XoYRn2XlzNVTNJgHys+fSLX4RTlLDe3JvHBEb4ESQSk4gd2fKrDU1dIThwEmtFXukUxcfv8NEMihEgBEC93h2EmjGH2qD+hiG5ygiprDRClj8RgpsyuID2k+kII8wTHvWB9jlTDUv3Lp/cUeoUw6ATrMGv9/alaWnOJeVRJq4Zm9Yj8QpP88jVFhtgwHyJeaYA/vka5ZKvnpozrJPifAB0zLQZYT0HjWN2OuCvukKB2nVHmpUFPvhdZA+iMLyjMaakv7GSi8jU9rJntCfORQKWY9UU9l1NUOeHVjRLDHqCQsOdSwee9TeLeF1bWIVTUyhAdFyRksak9pUt0Lg/m3kMPyZ9kuiAx/dE+LcBBNsJ3rYboRdE+wvUhkmTkmHbshxUhgoHzRwjk3PO7h0IYnkmSzXddESiurvM5KkdXU7iiij/s9Y1Y/EGdiXA65xaXM/JSsOrnDUCUmX5Nsa1W2+W8qElqpKxiWXDI4B+DRtzu13zihAREFgkPwwjQbutRon9gV5FTNhnp5JWNAo=
```

Comment:
Maybe put these in the global env definition so this giant string doesn't need to be copied around:

https://docs.travis-ci.com/user/environment-variables/#global-variables

Reply:
I was thinking to only give these secret credentials to the builds requiring them. We could reuse the value to avoid copying it over?

Reply:
> I was thinking to only give these secret credentials to the builds requiring them.

Yeah, that does make sense. Up to you.

> We could reuse the value to avoid copying it over?

What do you mean by this? I'm not sure I understand how this would look.

Reply:
nm, I see you already added. This looks good to me.

Reply:
Apparently, does not look good to Travis.

Reply:
I reverted the change to share secrets between builds, Travis does not seem to handle this advanced YAML feature.

I'm open to suggestions regarding reuse of the secrets.